cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -16,62 +16,122 @@ include::modules/technology-preview.adoc[leveloffset=+2]
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1725848[*BZ#1725848*])
+== Known issues
+
+* When adding a disk to a virtual machine via the *Disks* tab in the web console,
+the added disk always has a `Filesystem` volumeMode, regardless of the volumeMode
+set in the `kubevirt-storage-class-default` ConfigMap.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753688[*BZ#1753688*])
+
+* After migration, a virtual machine is assigned a new IP address. However, the
+commands `oc get vmi` and `oc describe vmi` still generate output containing the
+obsolete IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1686208[*BZ#1686208*])
++
+** As a workaround, view the correct IP address by running the following command:
++
+----
+$ oc get pod -o wide
+----
+
+* The virtual machines wizard does not load for users without administrator
+privileges. This issue is caused by missing permissions that allow users to load
+network attachment definitions.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1743985[*BZ#1743985*])
++
+** As a workaround, provide the user with permissions to load the network attachment
+definitions.
++
+. Define `ClusterRole` and `ClusterRoleBinding` objects to the YAML configuration
+file, using the following example:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+ name: cni-resources
+rules:
+- apiGroups: ["k8s.cni.cncf.io"]
+ resources: ["*"]
+ verbs: ["*"]
++
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <role-binding-name>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cni-resources
+subjects:
+- kind: User
+  name: <user to grant the role to>
+  namespace: <namespace of the user>
+----
++
+. As a `cluster-admin` user, run the following command to create the `ClusterRole`
+and `ClusterRoleBinding` objects you defined:
++
+----
+$ oc create -f <filename>.yaml
+----
+
+* When navigating to the *Virtual Machines Console* tab, sometimes no content
+is displayed. As a workaround, use the serial console.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753606[*BZ#1753606*])
+
+* When you attempt to list all instances of the {CNVProductName} operator from a
+browser, you receive a 404 (page not found) error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1757526[*BZ#1757526*])
++
+** As a workaround, run the following command:
++
+----
+$ oc get pods -n openshift-cnv | grep operator
+----
+
+* Some resources are improperly retained when removing {CNVProductName}. You
+must manually remove these resources in order to reinstall {CNVProductName}.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1712429[*BZ#1712429*]),
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1757705[*BZ#1757705*])
++
+** As a workaround, follow this procedure:
+link:https://access.redhat.com/articles/4484961[Removing leftover resources from container-native virtualization 2.1 uninstallation]
```

Comment:
I get an `access denied` from this link.

Reply:
Good catch! But I believe we're waiting to publish this article until we push the 2.1 docs. Is that correct, @aburdenthehand?

Reply:
It works for me. One is supposed to log in to access that page =)

Reply:
Oof, thought I was already logged in. Works for me, too!